web-animations-1/Overview.bs

```diff
@@ -924,6 +929,32 @@ matching condition from below:
 
 </div>
 
+### The current phase of an animation ### {#the-current-phase-of-an-animation}
+
+<a>Animations</a> provide a <a>timeline phase</a> to their <a>associated
+effect</a> called the animation's <dfn>current phase</dfn>.
+
+The <a>current phase</a> is calculated from the first matching condition from
+below:
+
+<div class="switch">
+
+:   If the animation's <a>hold phase</a> is <a lt="unresolved">resolved</a>,
+::  The <a>current phase</a> is the animation's <a>hold phase</a>.
+
+:   If <em>either</em> of the following are true:
+
+    1.  the animation has no associated <a>timeline</a>, or
+    2.  the animation's [=start time=] is <a>unresolved</a>.
```

Comment:
I'm curious about what this second condition is needed for?

Reply:
Great catch :D That was left over for a copy/paste. I am removing it.

Reply:
Turns out that the second condition is actually needed. It makes sure that while the animation is idle, the current phase is correctly "Inactive". I'll be adding it back in.